### ISSUE TITLE
Index controller thumbstick mapped to secondary2DAxis instead of primary2DAxis

### ISSUE THREAD

**Issue by halfpasttense** [ValveSoftware/steamvr_unity_plugin#745]
The Index controller thumbstick should be mapped to primary2DAxis like every other VR controller with a thumbstick. Having it map to secondary2DAxis will just result in more work for developers who will have to make the Index controller a special case.

**Comment by AVOlight** [comment]
are you using it this way?
```
device.TryGetFeatureValue(CommonUsages.primary2DAxis, out Vector2 vector);
device.TryGetFeatureValue(CommonUsages.grip, out float pull);
device.TryGetFeatureValue(CommonUsages.gripButton, out bool pressed);

```

**Comment by halfpasttense** [comment]
Yes using the new XR plugin, the primary2DAxis maps to the touchpad and the secondary2DAxis maps to the thumbstick on the Index controllers. Confirmed both through code and through the XR Interaction Debugger. It should definitely be the other way around.

**Comment by frederickjjoubert** [comment]
Not the same issue but a similar problem is that the knuckles B button is bound to the primary button, and the A button is bound to the grip. Verified using the Unity XR Interaction debugger.

**Comment by hookmanuk** [comment]
Can confirm, I have the same issue that the joystick binds to the secondary2daxis by default when running via unity.
